client/my-sites/plan-features-comparison/header.jsx

```diff
@@ -57,6 +68,32 @@ export class PlanFeaturesComparisonHeader extends Component {
 		);
 	}
 
+	renderPlanHeaderMarketplace() {
+		const { planType, title, translate, current } = this.props;
+
+		const headerClasses = classNames(
+			'plan-features-comparison__header',
+			getPlanClass( planType )
+		);
+
+		return (
+			<span>
+				{ current && <PlanPill isInSignup>{ translate( 'Your current plan' ) }</PlanPill> }
+				{ ( planType === PLAN_BUSINESS || planType === PLAN_BUSINESS_MONTHLY ) && (
+					<PlanPill isInSignup>{ translate( 'Ideal for you' ) }</PlanPill>
+				) }
+				<header className={ headerClasses }>
+					<h4 className="plan-features-comparison__header-title">{ title }</h4>
+				</header>
+				<div className="plan-features-comparison__pricing">
+					{ this.renderPriceGroup() }
+					{ /* { this.getBillingTimeframe() } */ }
+				</div>
+				{ /* { this.getAnnualDiscount() } */ }
```

Comment:
Do we need to disable these? Not sure what they do, wondering how much of this we can use as is.

Reply:
I was just experimenting with it at that time 😅